API.md

```diff
@@ -1207,11 +1208,14 @@ curl http://localhost:<port>/irods-http-api/<version>/data-objects \
     --data-urlencode 'new-data-size=<integer>' \
     --data-urlencode 'new-data-status=<string>' \
     --data-urlencode 'new-data-type-name=<string>' \
-    --data-urlencode 'new-data-version=<string>'
+    --data-urlencode 'new-data-version=<string>' \
+    --data-urlencode 'new-data-access-time=<string>' # Must be 11 bytes in length and padded with leading zeros.
```

Comment:
should we consider NOT requiring the clients to know about our non-standard storage of 11 bytes padded with leading zeroes?   can we just take in a normal number-as-string and do the padding ourselves in the HTTP API before sending to iRODS?

is that more confusing since we return the raw database value when someone asks for times from the database?

Reply:
Yes, we can do that. We need to determine if there are any downsides to that.

Consider the case where a user receives a raw timestamp from the catalog and attempts to pass it to the API. The user would have to adjust the timestamp. The HTTP API is now also working with different formats for timestamps (presentation vs input).

Reply:
right - so perhaps we just make the HTTP API accept 'both'?       we can detect if it's not 11-bytes, and pad if necessary?    maybe that's too much magic...